<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RouteBrowserPrefWidget</class>
 <widget class="QWidget" name="RouteBrowserPrefWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>457</width>
    <height>383</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Route Browser</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="0">
      <widget class="QCheckBox" name="mWaypointLabelCheckbox">
       <property name="text">
        <string>Populate Waypoint Labels By Default</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QCheckBox" name="mRouteNameCheckbox">
       <property name="text">
        <string>Populate Route Name By Default</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QCheckBox" name="mShowGlobalRoutesCheckbox">
       <property name="text">
        <string>Show Global Routes On Map Display</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="mWaypointLabelLineEdit">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Default Waypoint Label Prefix</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="mRouteNameLineEdit">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Default Route Name Prefix</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>mRouteNameCheckbox</tabstop>
  <tabstop>mRouteNameLineEdit</tabstop>
  <tabstop>mWaypointLabelCheckbox</tabstop>
  <tabstop>mWaypointLabelLineEdit</tabstop>
  <tabstop>mShowGlobalRoutesCheckbox</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
